infra/src/dsl/xroad.ts

```diff
@@ -237,3 +237,13 @@ export const PaymentSchedule = new XroadConf({
     PAYMENT_SCHEDULE_PASSWORD: '/k8s/api/PAYMENT_SCHEDULE_PASSWORD',
   },
 })
+
+export const CriminalRecord = new XroadConf({
+  env: {
+    XROAD_CRIMINAL_RECORD_PATH: {
+      dev: 'r1/IS-DEV/GOV/10005/Logreglan-Protected/Sakavottord-PDF-v2',
+      staging: '',
+      prod: '',
```

Comment:
If the intention is to specify that the values are missing please use the `MissingSetting` value. That will stop the build from merging to `main` since anyone can decide to take the deployment to Staging.